lib/ansible/modules/network/cnos/cnos_backup.py

```diff
@@ -259,9 +259,8 @@ def main():
     # cnos.debugOutput(path[0])
     if not os.path.exists(path[0]):
         os.makedirs(path[0])
-    file = open(outputfile, "a")
-    file.write(output)
-    file.close()
+    with open(outputfile, "a") as fh:
```

Comment:
```suggestion
    with open(outputfile, "a") as file:
```
It is a less intrusive change if you keep the original handle's name.

Reply:
OK. Changed. I'm getting warnings that `file` shadows a built-in; I had in mind if I were changing the line anyway, changing the variable name wouldn't matter.